LayoutTests/imported/w3c/web-platform-tests/css/css-color-hdr/parsing-expected.txt

```diff
@@ -25,6 +25,6 @@ PASS e.style['dynamic-range-limit'] = "dynamic-range-limit-mix(low, high, 10%)"
 PASS e.style['dynamic-range-limit'] = "dynamic-range-limit-mix(high 101%, standard 1%)" should not set the property value
 PASS e.style['dynamic-range-limit'] = "dynamic-range-limit-mix(high -1%, standard 1%)" should not set the property value
 PASS e.style['dynamic-range-limit'] = "dynamic-range-limit-mix(standard, high, 0.1)" should not set the property value
-PASS e.style['dynamic-range-limit'] = "dynamic-range-limit-mix(high 0%, standard 0%)" should not set the property value
-PASS e.style['dynamic-range-limit'] = "dynamic-range-limit-mix(dynamic-range-limit-mix(high 1%, standard 2%) 3%, dynamic-range-limit-mix(constrained-high 0%, high 0%) 6%)" should not set the property value
+FAIL e.style['dynamic-range-limit'] = "dynamic-range-limit-mix(high 0%, standard 0%)" should not set the property value assert_equals: expected "" but got "dynamic-range-limit-mix(high 0%, standard 0%)"
+FAIL e.style['dynamic-range-limit'] = "dynamic-range-limit-mix(dynamic-range-limit-mix(high 1%, standard 2%) 3%, dynamic-range-limit-mix(constrained-high 0%, high 0%) 6%)" should not set the property value assert_equals: expected "" but got "dynamic-range-limit-mix(dynamic-range-limit-mix(high 1%, standard 2%) 3%, dynamic-range-limit-mix(constrained-high 0%, high 0%) 6%)"
```

Comment:
Do we know why the spec forbids this?

Reply:
My guess is that "normalizing the sum to 100%" is done by dividing by the sum, so if the sum is zero that would be a division by zero.
And special casing 0 would not be obvious: Should `dynamic-range-limit-mix(high 0%, standard 0%)` be equivalent to high=standard=50%, or to high=standard=constrained-high=33%?

Reply:
There is an open question about how this works with calc so I left it be for now.

Reply:
The spec simply states that "If the sum of all percentages is 0%, then the function is invalid".

That said, its not clear how this works with calc() so I left it for the moment.

Reply:
Filed here -> https://github.com/w3c/csswg-drafts/issues/11678